src/unity/lib/visualization/summary_view.cpp

```diff
@@ -54,12 +54,7 @@ std::shared_ptr<transformation_output> summary_view_transformation::get() {
   std::shared_ptr<transformation_output> output =  m_transformers[m_index] -> get();
   ret.push_back(output);
 
-  auto fused_out = std::make_shared<summary_view_transformation_output>(ret);
-
-  fused_out->m_column_names = m_column_names;
-  fused_out->m_column_types = m_column_types;
-  fused_out->m_size = m_size;
-  fused_out->m_index = m_index;
+  auto fused_out = std::make_shared<summary_view_transformation_output>(summary_view_transformation_output(ret, m_column_names, m_column_types, m_size, m_index));
```

Comment:
You don't need to call the `summary_view_transformation_output` constructor explicitly - that will make a copy and then invoke the copy constructor in make_shared. You should be able to just pass the parameters directly to make_shared.